planning/behavior_path_planner/src/utils/avoidance/utils.cpp

```diff
@@ -675,21 +675,62 @@ void filterTargetObjects(
       o.overhang_lanelet = overhang_lanelet;
       lanelet::BasicPoint3d overhang_basic_pose(
         o.overhang_pose.position.x, o.overhang_pose.position.y, o.overhang_pose.position.z);
+
       const bool get_left = isOnRight(o) && parameters->enable_avoidance_over_same_direction;
       const bool get_right = !isOnRight(o) && parameters->enable_avoidance_over_same_direction;
+      const bool get_opposite = parameters->enable_avoidance_over_opposite_direction;
 
-      const auto target_lines = rh->getFurthestLinestring(
-        overhang_lanelet, get_right, get_left,
-        parameters->enable_avoidance_over_opposite_direction);
+      const auto merge_two_line = [](const auto & a, const auto & b) {
+        lanelet::Points3d new_points;
+        for (const auto & p : a) {
+          new_points.emplace_back(lanelet::utils::getId(), p.basicPoint());
```

Comment:
memo:
lane id is not needed to calculate distance. so just push the dummy initial id.